opendatahub-io/data-science-pipelines-operator
# Add new codeflare resource permissions to runner.

## The issue resolved by this Pull Request:
Resolves https://github.com/opendatahub-io/data-science-pipelines-operator/issues/327

## Description of your changes:
Adding new resource permissions as per #327

## Testing instructions
1. Deploy DSPO/DSPA
2. Deploy DW
3. Run a pipeline that utilizes codeflare resources
4. Confirm no rbac permission issues are encountered by the pipeline runner SA

## Checklist
